crosswalks/Java (Maven).csv

```diff
@@ -1,30 +1,30 @@
 Property,Java (Maven)
-codeRepository,scm / repositories
+codeRepository,scm
 programmingLanguage,
 review,
 runtimePlatform,
 targetProduct,
 applicationCategory,
 applicationSubCategory,
-downloadUrl,
+downloadUrl,distributionManagement

```

Comment:
Should be `distributionManagement/downloadUrl` according to https://maven.apache.org/pom.html#distribution-management

Reply:
Are there docs on how these crosswalks are intended to handle mappings to nested metadata? Since these crosswalks aren't even machine actionable I figured it'd be easier to point at the parent element.

I don't have the bandwidth to devote to refining these further, so if anyone else wants to contribute those changes please feel free!

Reply:
Not really, but in #279 I (dictatorially) decided to use an XPath-like syntax given the somewhat complex matching requirements, and I figure we might as well be consistent with that decision here.

> Since these crosswalks aren't even machine actionable

At Software Heritage we kind of use machine-read them: https://gitlab.softwareheritage.org/swh/devel/swh-indexer/-/blob/master/swh/indexer/metadata_dictionary/maven.py

Though of course there's a lot of Python code needed to validate and re-shape the data, but the XPath format may help

Reply:
`license` should probably go to `licenses / license` or `licenses / license / url` then?

https://maven.apache.org/pom.html#licenses

Reply:
`licenses / license / url`, yes